spec/README.md

```diff
@@ -14,7 +14,7 @@ and how they are communicated over the network.
 
 If you find discrepancies between the spec and the code that
 do not have an associated issue or pull request on github,
-please submit them to our [bug bounty](https://cometbft.com/security)!
+please submit them to our [bug bounty](https://github.com/cometbft/cometbft#security)!
```

Comment:
I guess this is the correct link:
```suggestion
please submit them to our [bug bounty](https://github.com/cometbft/cometbft/blob/v0.34.x/SECURITY.md)!
```
since we're using 0.34.x

Reply:
> I guess this is the correct link:
> 
> since we're using 0.34.x

Hi @rach-id , thanks for the suggestion. The link to bug bounty is updated now according to your suggestion